cmd/uhc/get/cmd.go

```diff
@@ -72,11 +72,36 @@ func init() {
 
 func run(cmd *cobra.Command, argv []string) {
 	// Check that there is exactly one command line parameter:
-	if len(argv) != 1 {
-		fmt.Fprintf(os.Stderr, "Expected exactly one argument\n")
+	if len(argv) < 1 || len(argv) > 2 {
+		cmd.Usage()
+		fmt.Fprintf(os.Stderr, cmd.Usage().Error())
 		os.Exit(1)
 	}
-	path := argv[0]
+	preParsePath := argv[0]
+	path := preParsePath
+
+	switch preParsePath{
+
+	// list resources
+
+	case "accounts", "accts":
+		path = "/api/accounts_mgmt/v1/accounts"
+	case "subscriptions", "subs":
+		path = "/api/accounts_mgmt/v1/subscriptions"
+	case "organizations", "orgs":
+		path = "/api/accounts_mgmt/v1/organizations"
+
+	// get individual resources
```

Comment:
Same comment regarding the style of comments.

Reply:
done